### PR TITLE
fix(face-landmarks): work only when one face is detected

### PR DESCRIPTION
**Pull request by gabiborlea**
Face centering and expression work only if one face is detected.
Remove previously added config flag for max faces detected.

### CODE REVIEW

File: react/features/face-landmarks/FaceLandmarksHelper.ts [hunk 1]
```diff
@@ -149,8 +142,12 @@ export class HumanHelper implements FaceLandmarksHelper {
     }
 
     getFaceExpression({ detections }: Detection): string | undefined {
+        if (!detections.length || detections.length > 1) {
+            return;
+        }
+
         if (detections[0]?.emotion) {
```
Comment (quitrk) [inline react/features/face-landmarks/FaceLandmarksHelper.ts:149]: you already checked above there's an item in the array

**Comment by saghul** [comment]
I'm a bit confused. I thought face-centrering worked with multiple faces and we computed a recatngle covering them all. Is this no longer the case? If not, why not?

**Comment by gabiborlea** [comment]
We discussed and the conclusion was to have face centering only with one face.

File: react/features/face-landmarks/FaceLandmarksHelper.ts [hunk 2]
```diff
@@ -178,30 +181,35 @@ export class HumanHelper implements FaceLandmarksHelper {
...
                 detections = await this.getDetections(image);
             }
 
-            if(detections) {
-                faceBox = this.getFaceBox({
-                    detections,
-                    threshold
-                });
-            } 
+            if ( this.getFaceCount(detections) > 1 ) {
```
Comment (saghul) [inline react/features/face-landmarks/FaceLandmarksHelper.ts:192]: style: unnecessary space after parens

**Comment by gabiborlea** [comment]
@saghul I filter the detection in the getDetections function, so The getFaceCount will receive only accurate detections

**Review by saghul — APPROVED** [review]

**Pull request merged**